Specification/3._Data_structure_description_language.md

```diff
@@ -4,36 +4,22 @@ permalink: /Specification/3._Data_structure_description_language/
 
 # Data structure description language
 
-The data structure description language (DSDL) is used to define data structures for exchange via the CAN bus.
-Every data structure is defined in a separate DSDL definition file.
-DSDL definition is then used to automatically generate the serialization/deserialization codes for a certain
-programming language.
-The tool that generates source codes from DSDL definition files is called the *DSDL compiler*.
+The Data Structure Description Language (DSDL) is used to define data structures for exchange via the CAN bus.
+The DSDL definitions are used to automatically generate the message serialization/deserialization code for a certain
+programming language. 
+The tool that generates source code from DSDL definition files is called the *DSDL compiler*.
 
 ## File hierarchy
+ 
+Each DSDL definition file specifies exactly one data structure that can be used for message broadcasting or a pair of structures that can be used for service invocation data exchange. 
```

Comment:
Excellent. The old wording was a remnant of a very early RFC document that used to be updated and rewritten left and right, so eventually it became a bit incoherent at places. You are probably going to find similar blunders in the following chapters of the specification. Thanks for fixing that.